client/blueprints/task/files/lib/__name__/app/models/__name__-task.js

```diff
@@ -0,0 +1,4 @@
+import DS   from 'ember-data';
+import Task from 'tahi/models/task';
+
+export default Task.extend({});
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.